### ISSUE TITLE
Configure order for script, template, style tags and auto format on save.

### ISSUE THREAD

**Issue by VasilVerdouw** [VasilVerdouw/vue-colors#2]
Based on:

https://www.reddit.com/r/vuejs/comments/1gxmbjk/comment/lyi4y71/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by jimp100** [comment]
This is functionality that many people already achieve with formatting/linting extensions like ESLint, (or possibly Prettier).  I would suggest implement this as disabled by default, to prevent clashing formatting operations running concurrently.

**Comment by VasilVerdouw** [comment]
> This is functionality that many people already achieve with formatting/linting extensions like ESLint, (or possibly Prettier).  I would suggest implement this as disabled by default, to prevent clashing formatting operations running concurrently.

I agree, I see this as low priority because many formatters probably already include this functionality. Disabled by default is also a good idea.
